src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__review">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <p class="card__review-reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-label">
+          Price:
+        </p>
+
+        <p class="card__price-value">
+          $2,199
+        </p>
+      </div>
+
+      <a href="#"
+        class="card__buy-button"
+        data-qa="hover"
+      >
+        Buy
+      </a>
```

Comment:
code is not aligned
```suggestion
      <a 
        href="#"
        class="card__buy-button"
        data-qa="hover"
      >
        Buy
      </a>
```
checklist 
![Screenshot 2023-10-03 at 13 08 26](https://github.com/mate-academy/layout_product-cards/assets/64746943/92cd6c14-28dc-4ccd-87a5-0029e33520fc)